CFC_DataCollector/recommender/recommendation_pipeline.py

```diff
@@ -14,26 +15,24 @@ def retrieve_alternative_trips(self, trip_list):
         return []
 
     def get_selected_user_utility_model(self, user_id):
-        return []
+        return UserUtilityModel.find_from_db(user_id)
 
     def recommend_trips(self, trip_id, utility_model):
         return []
```

Comment:
To build the user model, we obtain all trips and their alternatives. If we
fit on all of them together, we can learn bad things. This is what we have,
roughly:
Data:
[  (trip, [alternatives]), (trip2, [alternatives2]), .... ]
So if we fit on all of these together, we have a problem, because we could
be learning things like: "User chose alternative2-walking instead if
trip1because..."

On Wednesday, April 29, 2015, shankari <notifications@github.com> wrote:

> In CFC_DataCollector/recommender/recommendation_pipeline.py
> <https://github.com/e-mission/e-mission-server/pull/111#discussion_r29348302>
> :
>
> > @@ -14,26 +15,24 @@ def retrieve_alternative_trips(self, trip_list):
> >          return []
> >
> >      def get_selected_user_utility_model(self, user_id):
> > -        return []
> > +        return UserUtilityModel.find_from_db(user_id)
> >
> >      def recommend_trips(self, trip_id, utility_model):
> >          return []
>
> What specifically is the issue in the model building phase?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/e-mission/e-mission-server/pull/111/files#r29348302>.
>